internal/services/ingest.go

```diff
@@ -42,6 +49,7 @@ func NewIngestService(
 	tssRouter tssrouter.Router,
 	tssStore tssstore.Store,
 ) (*ingestService, error) {
+	log.SetLevel(log.InfoLevel)
```

Comment:
Was this line mean to be here?

Reply:
Actually I added this during one of the earlier iterations of this code. But its no longer needed. Good catch. Will remove it